README.MD

```diff
@@ -119,6 +119,7 @@ render(
 | gridType            | oneOf(['grid', 'tree']) | whether the grid will be a flat list or a tree view |
 | data                | arrayOf(object)         | local data for grid to display, [more information available](https://github.com/bencripps/react-redux-grid/blob/master/docs/PROVIDING_DATA.md)|
 | dataSource          | func                    | function which returns data to display, [more information available](https://github.com/bencripps/react-redux-grid/blob/master/docs/PROVIDING_DATA.md)|
+| rowRenderer         | func                    | function which returns the row contents for this row, [more information available](https://github.com/bencripps/react-redux-grid/blob/master/docs/USING_CUSTOM_ROW_RENDERERS.md)|
```

Comment:
Inside of row, where you're checking to see if a `renderer` is available, add this check:

```js
let rowEl;

if (!isPluginEnabled(plugins, 'ROW') 
	&& typeof plugins.ROW.renderer === 'function') {
	// super important that we pass rowProps and cells
	// since the user is almost certainly going to want both
	// lets make sure this gets documented
	rowEl = plugins.row.renderer({ rowProps, cells });
}

else {
	rowEl = (
	  <tr { ...rowProps }>
            { cells }
       </tr>
    );
}


// still get drag and drop even with custom row renderer
if (dragAndDrop) {
    return connectDragSource(connectDropTarget(rowEl));
}

return rowEl;
```